mobile-browser-based-version/src/Task Definition/mnist.js

```diff
@@ -12,30 +12,16 @@ export class MnistTask {
      * 
      * @returns Returns a tf.model or null if there is no model
      */
-    async get_model_from_storage() {
-        let model = await tf.loadLayersModel('indexeddb:://working_'.concat(training_information.model_id))
+     async get_model_from_storage() {
+        let model = await tf.loadLayersModel(training_information.save_path_db)
```

Comment:
Shouldn't this be `this.training_information` or am I missing something?

Reply:
Both ways are right, since the `training_information`object is outside of the class. But then when we initialise the class we also create a `this.training_information`object out of the external one. Do you prefer that I use `this`?

Reply:
I think it's best if you are consistent and use `this` inside of the class, so if we refactor it in the future, the class will be independent of the outside context. Thanks!

Reply:
Sounds good! It's changed to always using `this`inside the class